nuxeo-retention-web/rollup.config.js

```diff
@@ -28,7 +28,7 @@ const GLOBALS = {
 const IGNORES = [/^@(nuxeo|polymer)\//];
 
 // Keep these imports
-const KEEP = ['@nuxeo/nuxeo-ui-elements/import-href.js'];
+const KEEP = ['@nuxeo/nuxeo-ui-elements/import-href.js', '@nuxeo/moment/moment.js'];
```

Comment:
we can talk about it, but it should work without adding the `'@nuxeo/moment/moment.js'` into `KEEP` const. I have tested it, where I removed it and it works fine.

Reply:
removed

Reply:
not super friendly but you can't simply remove it from `KEEP` since it's used in JS code, you'd need to at least replace it with a dummy module import (by adding it to `IGNORES`) for your code to build, but then it wouldn't work.
One thing you could do it add moment to `GLOBALS`, probably somehting like  `'@nuxeo/moment': 'export moment'`; so it effectivelly reuses the already available moment.
That said, you can also just decide to keep it, like you did initially, this only means that we'd duplicate moment, since it'd be bundle in this addon too, but at least we ensure proper module isolation

Reply:
thanks @nelsonsilva for the suggestions! i'll try that out!

Reply:
I wasn't able to make it work so for the moment (pun intended) i'll leave it as is.